spec/lib/wiki_edits_spec.rb

```diff
@@ -58,6 +58,16 @@
     expect(User.first.wiki_token).to eq('invalid')
   end
 
+  it 'handles edit requests that are blocked' do
```

Comment:
These should also expect the BlockedEditsMonitor (or BlockedEditsReporter) to receive `:create_alerts_for_blocked_edits`, which tests the interaction between WikiResponse and the new alert code.